addons/io_scene_gltf2/__init__.py

```diff
@@ -672,7 +681,7 @@ def draw(self, context):
         layout.prop(operator, 'export_colors')
         layout.prop(operator, 'export_materials')
         col = layout.column()
-        col.active = operator.export_materials
+        col.active = operator.export_materials == "EXPORT"
```

Comment:
Per https://github.com/KhronosGroup/glTF-Blender-IO/issues/1189, could we move this to the *Include* section?

Reply:
Yes, this enum replaces the checkbox, that is already in "include" section in the mockup

Reply:
Yes, it's under the "include" section in the mockup, but not the code yet — let's make this change in code here, so that the new dropdown element is in its intended location from the beginning.